reports/blueprint/report.tex

```diff
@@ -183,16 +200,80 @@ \subsection{Validation}
 given by the target percentages in the hardware specifications.
 
 \section{Progress to Date} % Ethan / Tim for HW/SW respectively
-
+Since TeachEE is a project originally proposed in ELEC390, there is progress in
+the development of hardware and software.
```

Comment:
What is this supposed to mean. Perhaps better to just say something like "since the start of the term, the team has accomplished a few hardware and software development tasks"

Reply:
I was just afraid of using past tense. will fix